ring-alarm-mqtt.js

```diff
@@ -423,7 +423,7 @@ function initMqtt() {
 
 // Get Configuration from file
 try {
-    CONFIG = require('./config')
+    CONFIG = require('/data/options')
```

Comment:
```suggestion
    fs.access("/data/options.json", error => {
        if (!error) {
            // use hassio config, if found
            CONFIG = require('/data/options')
        } else {
            // use normal config
            CONFIG = require('./config')
        }
    });
```

I'm no no js expert, but might a fs check be more appropriate to not break compatibility?